test/box-tap/gh-6647-yield-gc-hook.test.lua

```diff
@@ -22,7 +22,7 @@ local function run_script(code)
     return res
 end
 
-local code = [[
+local lua_iproto_call = [[
```

Comment:
Nit: It is better to assign a name once rather than choose one and change it in a next commit.